src/bundle/Resources/public/scss/_typography.scss

```diff
@@ -37,3 +37,26 @@
         }
     }
 }
+
+.ez-adminsection-header {
+    border: none;
+
+    .card-header {
+        background-color: $ez-color-base-pale;
+
+        h2 {
+            margin-bottom: 0;
+            font-size: 1.125rem;
+            font-weight: 700;
+            color: $ez-black;
+        }
+    }
+
+    .card-body {
+        padding: .75rem 1.25rem;
+
+        h3 {
+            font-size: 1.0625rem;
```

Comment:
Shouldn't we use the same header sizing across the whole app? I'm not sure we should redefine it. We should be consistent. The same comment applies to `h2` selector above.